src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/EndpointSecurityComponentTest.kt

```diff
@@ -76,6 +78,7 @@ class EndpointSecurityComponentTest : ComponentTestBase() {
   private val returnedToCustodyRequest = ReturnedToCustodyRequest(OffsetDateTime.now().minusDays(1), OffsetDateTime.now().minusMinutes(1))
   private val createNoteRequest = CreateNoteRequest("subject", "details", FileName("filename"), "some content")
   private val stopRecallRequest = StopRecallRequest(StopReason.values().random())
+  private val recommendedRecallTypeRequest = RecommendedRecallTypeRequest(RecallType.values().random())
```

Comment:
`RecallType.values().random()` preferable to defining a `::RecallType.random()`?

Reply:
Not sure its worth doing that just for a test, but 🤷‍♂️